Marlin/src/feature/pause.cpp

```diff
@@ -107,25 +108,29 @@ void do_pause_e_move(const float &length, const float fr) {
 // public:
 
 bool move_away_flag = false;
+#if ENABLED(PAUSE_SPOOL_SWAP)
+bool swap_spool_enabled = false;
+int16_t swap_spool_stop=0;
+#endif
 
 bool pause_print(const float &retract, const float &z_lift, const float &x_pos, const float &y_pos,
                         const float &unload_length/*=0*/ , const int8_t max_beep_count/*=0*/, const bool show_lcd/*=false*/
 ) {
   if (move_away_flag) return false; // already paused
-
+  
   if (!DEBUGGING(DRYRUN) && (unload_length != 0 || retract != 0)) {
     #if ENABLED(PREVENT_COLD_EXTRUSION)
       if (!thermalManager.allow_cold_extrude &&
           thermalManager.degTargetHotend(active_extruder) < thermalManager.extrude_min_temp) {
         SERIAL_ERROR_START();
         SERIAL_ERRORLNPGM(MSG_TOO_COLD_FOR_M600);
-        return false;
+        return;
       }
-    #endif
-
-    ensure_safe_temperature(); // wait for extruder to heat up before unloading
+    #endif 
+    // wait for extruder to heat up before unloading
+    ensure_safe_temperature();
+    lcd_advanced_pause_show_message(ADVANCED_PAUSE_MESSAGE_STATUS);	
```

Comment:
i have placed this ensurance of temp just to have a security when pause start , not really usefull , but i prefer leave this option for one big reason
If pause appears just after a briding printing , the fan was at full speed and nozzle need to be heated , a little but need to be heated 
It's why i suggest to leave this , it's securised